addonscript/utils.py

```diff
@@ -0,0 +1,93 @@
+"""Common utilities for addonscript."""
+
+import time
+from uuid import uuid4
+
+from jose import jws
+from async_timeout import timeout
+
+
+def generate_JWT(context, _iat=None):
+    """Create and Sign a JWT valid for AMO HTTP requests.
+
+    Uses context.config 'jwt_user' and 'jwt_secret' and sets an expiration of
+    4 minutes (AMO supports a max of 5)
+    """
+    user = context.config['jwt_user']
+    secret = context.config['jwt_secret']
+    jti = str(uuid4())
+    iat = _iat or int(time.time())  # _iat for testing sanity
```

Comment:
Nit: `_iat` can be remove thanks to `freezegun`. It sets the current time returned by the std lib. Example: https://github.com/mozilla-releng/shipitscript/blob/master/shipitscript/test/test_ship_actions.py#L10